src/tools/test-targets.ts

```diff
@@ -68,21 +69,26 @@ export const pullTestTarget = async (
     return;
   }
 
-  writeYaml(data, options.destination);
+  const destination = await getPathToOctomindDir({ allowCreation: true });
+  if (!destination) {
+    throw new Error("No octomind directory found");
+  }
+  writeYaml(data, destination);
 
   console.log("Test Target pulled successfully");
 };
 
 export const pushTestTarget = async (
-  options: { testTargetId: string; source?: string } & ListOptions,
+  options: { testTargetId: string } & ListOptions,
```

Comment:
fyi, currently https://github.com/OctoMind-dev/automagically-action-execute/blob/v2.8.4/.github/workflows/main.yml#L126 this workflow relies on it being possible to configure a different source (so that the OTHER steps in that workflow do NOT use ymls)

We can solve that differently (by writing the ymls only in the step they should be used), but just so you are aware

Reply:
I don't get this comment, please come up with a solution in january ;) `so that the OTHER steps in that workflow do NOT use ymls`